html/changelogs/Incoming5643-slimechanges.yml

```diff
@@ -0,0 +1,6 @@
+author: Incoming5643
+
+delete-after: True
+
+changes: 
+  - rscadd: Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encoraged to investigate."
```

Comment:
Typo here btw. Encoraged = Encouraged.

If a typo in a changelog gets compiled in it can be obnoxious to change, so make sure to double check.